make

```diff
@@ -87,17 +113,17 @@ function images () {
   cp src/favicon.ico dist/favicon.ico
 
   while IFS= read -r -d '' FILE; do
-    CACHE_FILE="$CACHE_DIR/${FILE##"src/"}-$(md5 -q "$FILE")"
+    CACHE_FILE="$CACHE_DIR/${FILE##"src/"}-$(md5_hash "$FILE")"
     if ! [ -f "$CACHE_FILE" ]; then
       case ${FILE##*.} in
         "gif")
-          gifsicle --lossy --optimize=3 "$FILE" > "$CACHE_FILE"
```

Comment:
So, I locally switched to the [giflossy](https://github.com/kornelski/giflossy) fork for that feature.  Maybe we could feature detect on this?